.github/workflows/docker-build-push-dev.yml

```diff
@@ -0,0 +1,94 @@
+name: Build/push dev Docker images
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      pr-number:
+        description: "Pull Request Number"
+        required: true
+        default: "0"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: citydb-tool-dev
+  PLATFORMS: linux/amd64,linux/arm64
+
+jobs:
+  build-and-push-images:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Parse short sha
+        uses: benjlevesque/short-sha@v3.0
+        id: short-sha
+      - name: set lower case owner name
+        run: |
+          echo "OWNER_LC=${OWNER,,}" >>${GITHUB_ENV}
+        env:
+          OWNER: "${{ github.repository_owner }}"
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Log in to the Github Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: ${{ env.PLATFORMS }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: ${{ env.PLATFORMS }}
+      - name: Extract metadata (tags, labels) for docker image
+        id: meta
+        uses: docker/metadata-action@v5
+        env:
+          DOCKER_METADATA_ANNOTATIONS_LEVELS: manifest,index
+        with:
+          images: |
+            ${{ env.REGISTRY }}/${{ env.OWNER_LC }}/${{ env.IMAGE_NAME }}
+          flavor: |
+            latest=false
+          tags: |
+            # Latest tag for PR, PR number only
+            type=ref,event=pr,prefix=citydb-tool-pr-
+            # Tag for with commit sha appended
+            type=ref,event=pr,prefix=citydb-tool-pr-,suffix=-{{sha}}
+          labels: |
+            maintainer=Bruno Willenborg
+            maintainer.email=b.willenborg(at)tum.de
```

Comment:
Sure you don't want to adapt your email address and the maintainer organization?